robot/web/stream/server.go

```diff
@@ -50,7 +50,6 @@ type Server struct {
 // NewServer returns a server that will run on the given port and initially starts with the given
 // stream.
 func NewServer(
-	streams []gostream.Stream,
 	robot robot.Robot,
 	logger logging.Logger,
 ) (*Server, error) {
```

Comment:
Since this is a breakign change, can you check if goutils or app is using this function?

Reply:
Update: We checked through goutils and app and there does not seem to be any stream server `NewServer` refs.